src/errors/InvalidMustSupportError.ts

```diff
@@ -0,0 +1,10 @@
+import { Annotated } from './Annotated';
+
+export class InvalidMustSupportError extends Error implements Annotated {
+  specReferences = ['http://hl7.org/fhir/R4/profiling.html#mustsupport'];
```

Comment:
Is https://www.hl7.org/fhir/elementdefinition.html#interpretation another good reference to have for why this is disallowed?

Reply:
Agree... will add